frontend/src/components/messstelle/charts/MessstelleDiagramme.vue

```diff
@@ -36,14 +36,25 @@
       <!-- Inhalte -->
       <v-tabs-window-item :value="TAB_BELASTUNGSPLAN">
         <v-sheet
-          v-if="
-            belastungsplanDataDTO.ladeBelastungsplanMessquerschnittDataDTOList
-          "
           :max-height="contentHeight"
           width="100%"
           class="overflow-y-auto"
         >
+          <v-banner v-if="!hasSelectedVerkehrsarten">
+            <v-icon
+              color="error"
+              size="36"
+              icon="mdi-alert-decagram-outline"
+            />
+            <p class="ml-2">
+              Der Belastungsplan wird nicht angezeigt, da keine Verkehrsart
```

Comment:
Evtl ändern bzw. neutraler Formulieren:

Beispiel: "Bei fehlender Auswahl der Verkehrsart ist eine anzeige des Belastungsplan nicht mölgich."

Den Text mit dem oberen Text abstimmen: https://github.com/it-at-m/dave-frontend/pull/177/files#diff-114d1a0c6a0bb46bc814b3b71e0ea31ebd07fb8b2d4e206168976743c95f4ecbR172